pkg/azuredisk/nodeserver.go

```diff
@@ -351,3 +395,44 @@ func getFStype(attributes map[string]string) string {
 
 	return ""
 }
+
+func (d *Driver) findDiskAndLun(devicePath string) (string, int32, error) {
```

Comment:
add comments and example for this function

Reply:
done.